tests/js-tests/res/Shaders/example_Twist.vsh

```diff
@@ -0,0 +1,8 @@
+// http://www.cocos2d-iphone.org
+
+attribute vec4 a_position;
+
+void main()
+{
+    gl_Position = CC_MVPMatrix * a_position;
+}
```

Comment:
cpp-tests doesn't have these shader files, though it has shader file which has the same content but different file names.